src/main/charts/bitbucket/templates/NOTES.txt

```diff
@@ -13,4 +13,16 @@ To see the custom values you used for this release:
 
   $ helm get values {{ .Release.Name }} -n {{ .Release.Namespace }}
 
+{{ if .Values.ingress.create -}}
+Service URL: {{ ternary "https" "http" .Values.ingress.https -}}://{{ .Values.ingress.host }}
```

Comment:
For a bit more detail can we also include the relevant product name i.e. 

`Bitbucket service URL:`